src/Compilers/Core/Portable/Symbols/TypedConstant.cs

```diff
@@ -72,11 +72,11 @@ public bool IsNull
         /// <summary>
         /// The value for a non-array constant.
         /// </summary>
-        public object? Value
+        public object Value
```

Comment:
>public object Value [](start = 8, length = 19)

Isn't this `null` for the constant `null`?

Reply:
Thanks. I'd misread the assertion in constructor.

---
In reply to: [441893234](https://github.com/dotnet/roslyn/pull/44266#discussion_r441893234) [](ancestors = 441893234)